weather_sp/README.md

```diff
@@ -75,46 +78,79 @@ On the other hand, to specify a specific pattern use
 --input-pattern 'gs://test-tmp/era5/2017/*/*.nc'
 ```
 
-## Output
+## Output & Dimensions to split
 
 The base output file names are specified using the `--output-template` or `--output-dir` flags. These flags are mutually
-exclusive, and one of them is required.
+exclusive, and one of them is required. \
+The output formatting also specifies which dimensions to split by using Python formatting.
+For example, adding `{time}` in the output template or formatting will cause the file to be split along the time dimension.
```

Comment:
For a future PR: It would be cool if the user could choose the interval of time to split over.

Reply:
I've been thinking about this, and it's not easy given how netcfd and grib handle these differently. I'll have to look into that, but first I want to improve the splitting for netcdf to make it more flexible with different dimensions.